db/migrate/20170712033920_add_language_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddLanguageToUsers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :users, :language, :string
```

Comment:
We are advised not to use cookie to keep language preference, http://guides.rubyonrails.org/i18n.html

> 2.2.5 Storing the Locale from the Session or Cookies
> You may be tempted to store the chosen locale in a session or a cookie. However, do not do this. The locale should be transparent and a part of the URL. This way you won't break people's basic assumptions about the web itself: if you send a URL to a friend, they should see the same page and content as you. A fancy word for this would be that you're being RESTful. Read more about the RESTful approach in Stefan Tilkov's articles. Sometimes there are exceptions to this rule and those are discussed below.
>